tools/cli/workflowCommands.go

```diff
@@ -282,24 +282,34 @@ func startWorkflowHelper(c *cli.Context, shouldPrintProgress bool) {
 }
 
 func processSearchAttr(c *cli.Context) *commonpb.SearchAttributes {
-	rawSearchAttrKey := c.String(FlagSearchAttributesKey)
-	searchAttrKeys := trimSpace(strings.Split(rawSearchAttrKey, searchAttrInputSeparator))
+	sanitize := func(val string) []string {
```

Comment:
could be worth switching to multiple inputs in future, similar to here https://github.com/temporalio/temporal/pull/600

Reply:
good point but not now.